Source/Diagnostics/Diagnostics.cpp

```diff
@@ -54,10 +54,21 @@ Diagnostics::BaseReadParameters ()
     pp_diag_name.query("format", m_format);
     pp_diag_name.query("dump_last_timestep", m_dump_last_timestep);
 
+    amrex::ParmParse pp_geometry("geometry");
+    std::string dims;
+    pp_geometry.get("dims", dims);
+
     // Query list of grid fields to write to output
     bool varnames_specified = pp_diag_name.queryarr("fields_to_plot", m_varnames);
     if (!varnames_specified){
-        m_varnames = {"Ex", "Ey", "Ez", "Bx", "By", "Bz", "jx", "jy", "jz"};
+        if( dims == "RZ" )
+        {
+            m_varnames = {"Er", "Et", "Ez", "Br", "Bt", "Bz", "jr", "jt", "jz"};
+        }
+        else
+        {
```

Comment:
Just some style, we are not very consistent on those but often use this if inline:
```suggestion
        if( dims == "RZ" ) {
            m_varnames = {"Er", "Et", "Ez", "Br", "Bt", "Bz", "jr", "jt", "jz"};
        }
        else {
```